README.md

```diff
@@ -48,14 +48,65 @@ Setup development environment
 ```
 
 Config file
-------------------------------------
+----------------------------------------------------------------------------------------------------------------------
 `config.json` file should be located in the same directory as all source files. Sample configuration is included in `config.json.sample` file.
 
 Possible fields in config file:
-
+- `authUrl` - URL to keystone
+- `gohan` - gohan connection config, must contain object with following fields:
+  - `schema` - path to jsonschema
+  - `url` - URL to gohan server
+- `sidebar` - array of custom sidebar buttons, must contain objects with following fields:
+  - `path`
+  - `title` 
+  - `id`
+- `sidebarFavorites` - array of button id's to shows in the upper part of the sidebar
```

Comment:
`buttons' IDs` or `button's IDs`?
or 
`array of links to views, must contain resources' IDs`